roles/ovirt-host-deploy-firewalld/vars/firewalld_rules_gluster.yml

```diff
@@ -0,0 +1,32 @@
+host_deploy_firewalld_rules:
```

Comment:
Sahina, do all gluster services contain corresponding firewalld configuration file, so we can use service name instead of port numbers?
@sabose